# Differential Drive Controller Configurations

cmd_vel_publisher: "None"
odometry_subscriber: "None"

pid_velocity_controller_gain:
  p: 0
  i: 0
  d: 0

# Velocity and acceleration limits
# Whenever a min_* is unspecified, default to -max_*
limits:
  linear:
    x:
      max_velocity           : 1.1   # m/s
      max_acceleration       : 2.0   # m/s^2
  angular:
    z:
      max_velocity           : 1.1   # rad/s
      max_acceleration       : 3.0   # rad/s^2

